modules/37_industry/subsectors/postsolve.gms

```diff
@@ -63,5 +63,70 @@ pm_IndstCO2Captured(ttot,regi,entySe,entyFe,secInd37,emiMkt)$(
   / o37_demFeIndSub_SecCC(ttot,regi,secInd37);
 
 
+
+display vm_demFENonEnergySector.l;
+
+*** to be deleted before merge of feedstocks implementation, just checking the values
```

Comment:
If these parameters just serve as sanity check to see if your formulation works, they shouldn't be included in the final PR merge. 
A few years ago (at least 4) it was decided that output parameters should be kept to a minimum at the gams code to avoid code cluttering. Quite some work was done to move them outside of the GAMs code and to use the R reporting libraries instead with this intention. If you find these parameters useful, you should add them as a remind2 reporting function.